test/uk/gov/hmrc/taxenrolmentassignmentfrontend/controllers/views/ContactSASpec.scala

```diff
@@ -92,58 +92,43 @@ class ReportSuspiciousIDSpec extends TestFixture {
           .getElementsByClass(Selectors.summaryListValue)
           .text() shouldBe accountDetails.email.get
       }
-      "includes the last signed in field with correct value" in {
-        suspiciousIdDetailsRows
-          .get(2)
-          .getElementsByClass(Selectors.summaryListKey)
-          .text() shouldBe "Last signed in"
-        suspiciousIdDetailsRows
-          .get(2)
-          .getElementsByClass(Selectors.summaryListValue)
-          .text() shouldBe accountDetails.lastLoginDate
-      }
```

Comment:
Same here, this test shouldn't be deleted!

Reply:
depending on what is returned by mfa, at least one field of the mfa details ,the text message, is getting validated.
To ease the concern we are having a story for validating mfa details call and validating the returned value across the journey, so guess with in we can validate the permutations.